### ISSUE TITLE
[Accessibility] Reading order of the contributed custom extension's contents in it's pane header is illogical.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#228023]
-     VS Code Version: 1.88.0
-     OS Version: MacOS Sonoma (v14.6.1)

**Issue**:  There is an additional extension being contributed in side bar with Pane header title (expandable). But the contents of the extension's pane does not include the `.pane-body ` element that follows the expandable header. Instead, they are included in an iframe after all of the other code editor elements. This produces an illogical reading order.
So basically the section of `extension's pane header` is structured after the status bar which is illogical reading order as per A11y standards. 

**[Recommendation]**
Ensure that the reading order of content is logical. The best way to set the reading order is to reorder the elements in the DOM.

The accordion panel content should be structured after the accordion control.

**Comment by meganrogge** [comment]
> There is an additional extension being contributed in side bar with Pane header title (expandable)

I'm really confused by this. could you provide a screenshot?
